src/MessagingThreadListItem/MessagingThreadListItem.tsx

```diff
@@ -110,10 +119,11 @@ export const MessagingThreadListItem: React.FC<
           {subContent && (
             <FlexBox alignItems={ItemAlign.CENTER}>
               <div className={subContentClass}>{subContent}</div>
-              {indicator}
+              {timestamp && indicator}
```

Comment:
Why do we only want to show the indicator when a timestamp is around? :O

Reply:
So we have 3 options: either display timestamp + indicator, just timestamp, or just indicator. There is no instance of messaging using just an indicator yet, so this is a new use case for invites (mocks don't have timestamp, and we don't sort threads by order of time invites are sent out). With just timestamp, it looks fine to have it placed in the same horizontal axis of the title, but with just indicator, it looks a bit funky for it to be on the lower right of the thread list item. we want it vertically centered as another column of the overall parent flexbox on the right side of the thread item. Does that make sense?

Reply:
Ah yep, that clears things up! :)